### PR TITLE
chore: added keywords for plugins

### PR DESCRIPTION
**Pull request by shahednasser**
Added keywords for plugins that we want to be scraped and shown on the website.

### CODE REVIEW

File: packages/medusa-plugin-restock-notification/package.json [hunk 1]
```diff
@@ -38,5 +38,9 @@
     "express": "^4.17.1",
     "medusa-core-utils": "^1.1.31"
   },
-  "gitHead": "cd1f5afa5aa8c0b15ea957008ee19f1d695cbd2e"
+  "gitHead": "cd1f5afa5aa8c0b15ea957008ee19f1d695cbd2e",
+  "keywords": [
+    "medusa-plugin",
+    "medusa-plugin-notification"
```
Comment (olivermrbl) [inline packages/medusa-plugin-restock-notification/package.json:44]: todo: Not a notification plugin, but hard to know given the naming 😅 It's just a custom feature for sending out restock emails

File: packages/medusa-plugin-slack-notification/package.json [hunk 2]
```diff
@@ -43,5 +43,9 @@
     "medusa-test-utils": "^1.1.37",
     "moment": "^2.27.0"
   },
-  "gitHead": "c46300d58fbfd0b2dc2c02745ae143e6247e885b"
+  "gitHead": "c46300d58fbfd0b2dc2c02745ae143e6247e885b",
+  "keywords": [
+    "medusa-plugin",
+    "medusa-plugin-notification"
```
Comment (olivermrbl) [inline packages/medusa-plugin-slack-notification/package.json:49]: todo: Not a notification plugin either, but again hard to know 😅 This is Slackbot forwarding confirmed orders to a channel

File: packages/medusa-source-shopify/package.json [hunk 3]
```diff
@@ -47,5 +47,9 @@
     "eslint": "^7.32.0",
     "jest": "^26.6.3"
   },
-  "gitHead": "cd1f5afa5aa8c0b15ea957008ee19f1d695cbd2e"
+  "gitHead": "cd1f5afa5aa8c0b15ea957008ee19f1d695cbd2e",
+  "keywords": [
+    "medusa-plugin",
+    "medusa-plugin-migration"
```
Comment (olivermrbl) [inline packages/medusa-source-shopify/package.json:53]: suggestion: Should we name these `medusa-plugin-source` instead? Long term, I could imagine these being used for more than just migrations. We've been brainstorming on using these as connectors allowing you to have two-way sync. of products (and potentially more)

**Review by olivermrbl — APPROVED** [review]
Great, left a couple of comments! Feel free to merge after addressing them 👍

File: packages/medusa-plugin-restock-notification/package.json (hunk 1 above)
Comment (shahednasser) [reply packages/medusa-plugin-restock-notification/package.json:44]: Should we just leave it under "other" then?

File: packages/medusa-plugin-slack-notification/package.json (hunk 2 above)
Comment (shahednasser) [reply packages/medusa-plugin-slack-notification/package.json:49]: I thought that categorized it as a plugin 😅 Again what should it be for this one?

Comment (olivermrbl) [reply packages/medusa-plugin-slack-notification/package.json:49]: I think `medusa-plugin` should suffice. Similar to the wishlist plugin

Comment (olivermrbl) [reply packages/medusa-plugin-slack-notification/package.json:49]: For reference, a notification plugin will follow our notification-service interface. Same goes for payment, fulfillment, etc.

Comment (olivermrbl) [reply packages/medusa-plugin-slack-notification/package.json:49]: On second thought, I'm fine with keeping them in the notification grouping for now, so feel free to merge!

**Pull request merged**